core/templates/dev/head/pages/exploration_player/PlayerServices.js

```diff
@@ -338,15 +347,10 @@ oppia.factory('oppiaPlayerService', [
             StatsReportingService.recordStateTransition(
               oldStateName, newStateName, answer,
               LearnerParamsService.getAllParams());
-
-            if (exploration.isStateTerminal(newStateName)) {
-              StatsReportingService.recordExplorationCompleted(
-                newStateName, LearnerParamsService.getAllParams());
-            }
           }
 
           $rootScope.$broadcast('updateActiveStateIfInEditor', newStateName);
-          $rootScope.$broadcast('playerStateChange');
+          $rootScope.$broadcast('playerStateChange', newStateName);
```

Comment:
I'm a bit confused by this; we are currently broadcasting ``playerStateChange`` in both cases, I just added a listener for it. ``playerStateChange`` is also listened to by other parts of the exploration editor (``PreviewTab`` I believe). I can add the function call, but it's roughly the same as what's happening here, anyway.